pkg/infra/log/term/terminal_logger.go

```diff
@@ -14,14 +14,22 @@ import (
 	"github.com/grafana/grafana/pkg/infra/log/level"
 )
 
-const (
+var (
 	timeFormat     = "2006-01-02T15:04:05-0700"
 	termTimeFormat = "01-02|15:04:05"
-	floatFormat    = 'f'
-	termMsgJust    = 40
-	errorKey       = "LOG15_ERROR"
 )
 
+const (
```

Comment:
Would it make sense to keep the terminal output below as is when a TTY is attached vs not attached? 

Now when levels are changed this output will be
```
ERROR[04-06|12:34:52] Failed to load app dashboards            logger=plugindashboards error="open /home/marcus/go/src/github.com/grafana/grafana/data/plugins/grafana-iot-roci-app/dashboards/roci-main-dashboard.json: no such file or directory"
INFO[04-06|12:34:52] Updating app from configuration          logger=provisioning.plugins type=alexanderzobnin-zabbix-app enabled=true
DEBUG[04-06|12:34:52] NOTIFY_SOCKET environment variable empty or unset, can't send systemd notification logger=server
```

Rather than currently:
```
EROR[04-06|12:34:52] Failed to load app dashboards            logger=plugindashboards error="open /home/marcus/go/src/github.com/grafana/grafana/data/plugins/grafana-iot-roci-app/dashboards/roci-main-dashboard.json: no such file or directory"
INFO[04-06|12:34:52] Updating app from configuration          logger=provisioning.plugins type=alexanderzobnin-zabbix-app enabled=true
DBUG[04-06|12:34:52] NOTIFY_SOCKET environment variable empty or unset, can't send systemd notification logger=server
```

Which makes it harder to read/scan the output when not aligned.

Reply:
WDYT about having `%5s` for the levels and getting this:

```
ERROR[04-06|12:34:52] Failed to load app dashboards            logger=plugindashboards error="open /home/marcus/go/src/github.com/grafana/grafana/data/plugins/grafana-iot-roci-app/dashboards/roci-main-dashboard.json: no such file or directory"
INFO [04-06|12:34:52] Updating app from configuration          logger=provisioning.plugins type=alexanderzobnin-zabbix-app enabled=true
DEBUG[04-06|12:34:52] NOTIFY_SOCKET environment variable empty or unset, can't send systemd notification logger=server
```

Reply:
Sure, that should work :heart_eyes:

Reply:
Thanks folks :)